ClientLib/src/main/java/com/o3dr/services/android/lib/drone/companion/solo/tlv/SoloPanoOptions.java

```diff
@@ -0,0 +1,157 @@
+package com.o3dr.services.android.lib.drone.companion.solo.tlv;
+
+import android.os.Parcel;
+import android.support.annotation.IntDef;
+
+import java.nio.ByteBuffer;
+
+/**
+ * Created by phu on 6/24/16.
+ */
+public class SoloPanoOptions extends SoloShotOptions {
```

Comment:
@phmagic can you get @TShapinsky to take a look at updating the `ShotState` inside of the Solo app. This is a hack to get around a minor inconvenience, which i'd rather avoid, since the need for such hack might go away, but then it'll be very difficult to fix the architecture for this class.